### PR TITLE
Andrew/update dependencies

### PR DESCRIPTION
**Pull request by AndrewSepic**
- Update dependencies to reduce vulnerabilities.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,16 +17,16 @@
     "test": "jest --maxWorkers=4"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=18"
   },
   "devDependencies": {
     "eslint": "^4.18.2",
```
Comment (chriswhong) [inline package.json]: Should we update eslint?

File: package.json [hunk 2]
```diff
@@ -17,16 +17,16 @@
...
     "eslint-plugin-node": "^6.0.1",
-    "husky": "^0.14.3",
-    "jest": "^25.0.0",
+    "husky": "^8",
+    "jest": "^27.0.0",
     "lerna": "^2.9.0",
     "lint-staged": "^9.4.2",
```
Comment (chriswhong) [inline package.json]: We have updated lint-staged to v15 on other dependencies, any reason not to here?

**Review by chriswhong — APPROVED** [review]
LGTM, eslint and lint-staged could be updated to keep up with other mapbox packages but not critical.

File: package.json (hunk 1 above)
Comment (AndrewSepic) [reply package.json]: Yes,  Thanks for catching that.

Comment (AndrewSepic) [reply package.json]: Updated & pushed new updates.

**Pull request merged**